pro/src/screens/CollectiveOfferSummary/__specs__/CollectiveOfferSummary.spec.tsx

```diff
@@ -0,0 +1,115 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import { createBrowserHistory } from 'history'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router'
+import type { Store } from 'redux'
+
+import {
+  CollectiveOffer,
+  CollectiveOfferTemplate,
+  EducationalCategories,
+} from 'core/OfferEducational'
+import {
+  categoriesFactory,
+  subCategoriesFactory,
+} from 'screens/OfferEducational/__tests-utils__'
+import {
+  collectiveOfferStockFactory,
+  collectiveOfferTemplateFactory,
+  offerFactory,
+} from 'screens/OfferEducationalStock/__tests-utils__'
+import { configureTestStore } from 'store/testUtils'
+
+import CollectiveOfferSummary from '../CollectiveOfferSummary'
+
+const renderCollectiveOfferSummary = (
+  store: Store,
+  offer: CollectiveOfferTemplate | CollectiveOffer,
+  categories: EducationalCategories
+) => {
+  const history = createBrowserHistory()
+
+  render(
+    <Provider store={store}>
+      <Router history={history}>
```

Comment:
Tu n'utilise pas `history`, essai donc de t'en passer : 

```suggestion
  render(
    <Provider store={store}>
      <MemoryRouter>
```

Note: react-router-v6 n'utilisera plus `history.push()` mais `navigate()`:
* tentons de nous débarasser de nos utilisation de history
* n'ajoutons pas de nouvelles utilisation de history.

Si tu avait besoin de changer d'url, tu pourrait avoir : 

```js
render(
    <Provider store={store}>
      <MemoryRouter initialEntries={[myUrlFromProps]}>
```

Reply:
Yes c'est bien vu ! On pourrait l'ajouter dans les [guidelines](https://www.notion.so/passcultureapp/Guidelines-tests-front-pro-d196f07d7305483d91e23a27e11a1dc7) peut être :